nimble/host/audio/syscfg.yml

```
@@ -17,5 +17,8 @@
 #
 
 syscfg.defs:
+    BLE_AUDIO:
+        description: 'Indicates that a BLE Audio is present.'
```

I'd move this config to `host/syscfg.yml`, just like `BLE_MESH` is defined.
Plus, I'd suggest to describe this option like:

```suggestion
        description: 'This option enables Bluetooth LE Audio support.'
```

---

Fully agree :)